test/features/_walletLivingDocumentation/US-402-importWallet.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Verify the user can import an existing wallet
+
+	@US-402 @android @ios
+	Scenario: Verify the user can import an existing wallet
+		Given I launch the app
+		When I proceed to import an existing wallet
+		Then I input the master key of an existing wallet
```

Comment:
I think this scenario might be rewritten as follows:
` Given I proceed to import an existing wallet
   When I input the master key of an existing wallet
   Then I verify I am able to recover the existing wallet`
 
We should remove steps that do not add value to the test itself like the given step in this case ;)